README.md

```diff
@@ -51,3 +52,11 @@ Icons should be specified as a single word, case-insensitive
 - PostePay
 - UnionPay
 - Visa -- Aliases: `Delta`, `UKE`
+
+## Default Layouts
+
+Layouts specify number of icons per row.
+
+| 1 | 2 | 3 | 4 | 5 | 6 |
+|---|---|---|---|---|---|
+| [1] | [2] | [2, 1] | [2, 2] | [3, 2] | [3, 3] |
```

Comment:
I think I'd just document it like this instead:

```php
[
    1 => [1],
    2 => [2],
    3 => [2, 1],
    4 => [2, 2],
    5 => [3, 2],
    6 => [3, 3],
]
```

Github isn't rendering my suggested change properly because it includes ```php lol, but wrap it in backticks as a code block I think